docs/user/rstudio/ide/guide/publish/connecting.qmd

```diff
@@ -2,34 +2,46 @@
+![Global options menu](images/connecting-1.png){fig-alt="Global options menu"}
 
-In the Options panel that appears, choose **Publishing** in the sidebar, and use the **Connect\...** button to add a new account.
+You'll need to connect RStudio to the remote hosting environment (e.g. Posit Connect, Shinyapps.io, RPubs.com) and authorize RStudio to use your account. To add a publishing account to RStudio, press **Tools** in the main menu, then **Global Options\...**.
 
-![RStudio Connect account configuration](https://docs.rstudio.com/connect/user/images/screenshots/connecting-3.png){alt="RStudio Connect account configuration"}
+In the Options panel that appears, choose **Publishing** in the sidebar, and use the **Connect\...** button to add a new account.
 
-Select that you want to configure an RStudio Connect account, then enter the address of your server. Your Connect administrator can provide you with the address of the RStudio Connect server.
+![](images/connecting-publishing.png){fig-alt="The Global options menu, highlighting the Publishing tab and the Connect... button"}
 
-Clicking **Next** opens a login window for RStudio Connect.
 
-\
-![Link RStudio Connect to the IDE](https://docs.rstudio.com/connect/user/images/screenshots/loggedin.png){alt="Link RStudio Connect to the IDE"}
+Select the publishing destination, then follow the **Connect Account** wizard's instructions for that particular product.
 
-Log into Connect and click **Connect** to authorize the IDE to deploy on your behalf. Then return to the IDE and click the **Connect Account** button.
+![Publishing account configuration](images/connect-account.png){fig-alt="Publishing account configuration"}
 
-Once you have successfully configured the RStudio IDE, you may now publish content to RStudio Connect.
+Log in and click **Connect** to authorize the IDE to deploy on your behalf. Then return to the IDE and click the **Connect Account** button.
 
-If you are ready to publish content to RStudio Connect, continue to the [Publishing](https://docs.rstudio.com/connect/user/publishing/) chapter.
+Once you have successfully configured the connection within RStudio, you may now publish content to Posit Connect or Shinyapps.io.
 
-## Update the IDE
+For more information on managing or deploying content to Posit Connect, see the [Posit Connect User Guide](https://docs.posit.co/connect/user/).
 
-You will need to install at least version 1.0.44 of the RStudio IDE to interact with Connect. You can confirm this by opening the IDE and clicking "Help" \> "About RStudio" and checking the version number at the top of that pane. If you are not running at least version 1.0.44, you should [download the latest release](https://www.rstudio.com/products/rstudio/download/).
+If you are ready to publish content to RStudio Connect, continue to the [Publishing](https://docs.posit.co/connect/user/publishing/) chapter.
```

Comment:
^ Posit Connect here

By the way, is it settled that we are using the spiky hub logo for Posit Connect? We were getting some back and forth on that one, but I'm going to put in that PR if it's confirmed